### DISCUSSION TITLE
Redesigned download page is a worse user experience than the current one

### DISCUSSION THREAD

**Discussion by richardlau** [nodejs/nodejs.org#5337] [General]
I believe the plan is to port the redesigned https://nodejs.dev/en/download/ page here. It's a much worse user experience than the current https://nodejs.org/en/download.

Problems:
1. Downloads are now split into two sections, the three highlighted boxes for the installers and source code, with the rest of the downloads in a separated section right at the bottom of the page.
2. The middle of the page has an "Upcoming releases" section. I think this should be at the bottom of the page, or at least not interrupting the section with the download links.
3. The "Additional Downloads" are all behind collapsed sections that need to be expanded.
4. In "Additional Downloads", "AIX" is not "Linux".
5. No link to the official Docker image
6. No links to SHASUMS for the downloads.

If I wanted to download, e.g. the Linux x64 binary, today I can go to https://nodejs.org/en/download and click on the link directly. On the redesigned page I first need to scroll down and then expand the collapsed section before I can click on the link.

**Comment by bmuenzenmeyer** [comment]
I think we can close this now that the nodejs.dev design is dropped.
